packages/falter-berlin-autoupdate/files/autoupdate.sh

```diff
@@ -114,6 +118,11 @@ if [ "$UPTIME" -lt 7200 ]; then
     exit 2
 fi
 
+if echo "$FREIFUNK_RELEASE" | grep "snapshot"; then
```

Comment:
start with this if statement. we don't need to check for a newer stable release at all if we run on snapshot.

Reply:
Reordered in new version. I pretty much like the Idea doing autoupdates between 3 and 5 a.m. This change is in `post-inst.sh`